pkg/transformer/openshift/openshift_test.go

```diff
@@ -60,6 +62,15 @@ func TestInitDeploymentConfig(t *testing.T) {
 	if spec.Spec.Triggers[1].ImageChangeParams.ContainerNames[0] != "myfoobarname" {
 		t.Errorf("Expected myfoobarname for name, actual %s", spec.Spec.Triggers[1].ImageChangeParams.ContainerNames[0])
 	}
+
+	if spec.Spec.Template.Spec.Containers[0].Stdin != true {
```

Comment:
Can we create a separate test? Because, this test gets hidden and never shows on output as this was tested?